.github/workflows/publish-bicep.yaml

```diff
@@ -16,7 +16,7 @@
 
 name: Update extensibility provider types
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
nice 👍

Reply:
Can we leave a big scary "thar be dragons here" comment about the security assumptions of `pull_request_target`?

Something like:

```
# Please be careful making changes to this workflow. This uses pull_request_target so it has access to our 
# secrets. This is OK because we're just copying files out of the contributors PR branch. 
#
# DO NOT run any code from the pull-request including makefile targets.
#
# See the official docs for more info:  https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target
```